ote/src/cljs/ote/app/routes.cljs

```diff
@@ -47,6 +47,17 @@
 
 (defmethod on-navigate-event :default [_] nil)
 
+(defn- send-startup-events [event]
+  (let [e! (fn e! [event]
+             (binding [tuck/*current-send-function* e!]
```

Comment:
What is happening here?

Reply:
Tuck relies on a dynamic binding for *current-send-function* to be present (because the `e!` is not passed in to process event)

We need to bind it to the same function when processing so that it is available.